docs/getting-started/getting-started-with-services.rst

```diff
@@ -22,7 +22,7 @@ Create Your Service
 The first step is to create a file named ``.dockstore.yml``. The ``.dockstore.yml``  is a configuration file used to describe your service. Technically, having a valid YAML and the right version is
 all that is required for your service to appear on Dockstore. But, we will briefly cover an example ``.dockstore.yml`` written for the Xena Hub to show what will generally be needed to create a working service.
 You can also view a template ``.dockstore.yml`` file `here
-<https://github.com/dockstore/dockstore-documentation/blob/1e487775aa8ca22e38ec2193dc671d31b3d1eecf/docs/assets/templates/.dockstore.yml>`_.
+<https://github.com/dockstore/dockstore-documentation/blob/1.9.0-bucket/docs/assets/templates/.dockstore.yml>`_.
```

Comment:
`1.9.0-bucket` might not last too much longer when we merge it with develop, wonder how we should approach these

Reply:
(is there the equivalent of `this` or link to `../assets/templates/.dockstore.yml`)